src/connector.js

```diff
@@ -116,6 +117,18 @@ function initSearchUI() {
 	paramsDetect.isAdvancedSearch = !!document.getElementById( 'advseacon1' ) || winPath.endsWith( '/advanced-search.html' ) || winPath.endsWith( '/recherche-avancee.html' );
 	paramsDetect.enableHistoryPush = !paramsDetect.isAdvancedSearch;
 
+	// Auto detect relative path from originLevel3
+	if ( paramsOverride.originLevel3 ) {
+		try {
+			var regPath = /.+?:\/\/.+?(\/.+?)(?:#|\?|$)/;
+			var pathName = regPath.exec( paramsOverride.originLevel3 )[1];
+			paramsDetect.originLevel3RelativeUrl = pathName;
```

Comment:
This is more-so a comment:

Your regex seems to work fine, but it could've been simpler to use the URL constructor.  Something like:
```
let originURL = new URL( paramsOverride.originLevel3 );
paramsDetect.originLevel3RelativeUrl = originURL.pathname + originURL.search;
```
In the example above, I added `+ originURL.search` just as potential solution in case **originLevel3** contains parameters. But I don't know if parameters would ever be included in the originLevel3 at this point.